### ISSUE TITLE
The PSNR problem of HAT_SRx4.pth

### ISSUE THREAD

**Issue by Ironteen** [XPixelGroup/HAT#45]
I use the provided HAT_SRx4.pth and Set5 datasets, and run the 
`python hat/test.py -opt ./options/test/HAT_SRx4.yml`

The results are follows:

`2023-02-15 10:15:58,541 INFO: Loading HAT model from ./experiments/pretrained_models/HAT_SRx4.pth, with param key: [params_ema].
2023-02-15 10:15:58,767 INFO: Model [HATModel] is created.
2023-02-15 10:15:58,767 INFO: Testing Set5...
2023-02-15 10:16:00,483 INFO: Validation Set5
         # psnr: 28.6280        Best: 28.6280 @ HAT_SRx4 iter
         # ssim: 0.8403 Best: 0.8403 @ HAT_SRx4 iter`

The PSNR is much lower than the  results reported in the paper.
So how to reproduce the inference accuracy with the pretrained models

**Comment by chxy95** [comment]
@Ironteen You may check the correctness of your data. Torch 1.8 may also cause abnormal performance.

**Comment by Ironteen** [comment]
> @Ironteen You may check the correctness of your data. Torch 1.8 may also cause abnormal performance.

Thanks, I get the right results with torch 1.7. Is torch 1.9 or higher not available?

**Comment by chxy95** [comment]
@Ironteen Only torch 1.8 has this problem from my experience.

**Comment by fukka** [comment]
I am using 1.7.1+cu101, and I got the following which is lower than the paper result.
	 # psnr: 31.0627	Best: 31.0627 @ HAT_x4_SR_baseline_Set5 iter
	 # ssim: 0.9060	Best: 0.9060 @ HAT_x4_SR_baseline_Set5 iter
